src/status_im/ui/screens/wallet/address_add_edit/utils.cljs

```diff
@@ -0,0 +1,14 @@
+(ns status-im.ui.screens.wallet.address-add-edit.utils)
+
+(defn random-emoji
```

Comment:
is this needed?? 🤔 This is to get the initial random emoji right?
Perhaps you can align with @smohamedjavid on how to select it from the set he added in his emoji picker work?

Reply:
done